modules/navier_stokes/examples/solidification/tests

```diff
@@ -8,4 +8,11 @@
     requirement = 'The system shall be able to simulate the melting of galium.'
     heavy = true
   []
+  [galium_melting_physics]
+    type = RunApp
+    input = 'galium_melting-physics.i'
+    cli_args = "Nx=20 Ny=10"
+    requirement = 'The system shall be able to simulate the melting of galium using the Navier Stokes physics syntax.'
+    heavy = true
+  []
```

Comment:
you're just carrying-on the tradition, but it sure would be nice to test *something* other than just making sure the app runs

Reply:
try running the original input on your machine you'll see